packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -39,6 +47,7 @@ This config would govern chunking compressed batches only. We will not be enabli
 If the features are enabled using the configs, they can be disabled at runtime via feature gates as following:
 
 -   `Fluid.ContainerRuntime.CompressionDisabled` - if set to true, will disable compression (this has a side effect of also disabling chunking, as chunking is invoked only for compressed payloads).
+-   `Fluid.ContainerRuntime.DisableGroupedBatching` - if set to true, will disable grouped batching.
```

Comment:
Are we moving towards inverting the flag name convention used by the other two? `<Feature>Disabled -> Disable<Feature>`. If intentional moving forward, great; if not, can we still make it match?

Reply:
I don't think there's currently a convention around feature flag names.
@markfields Maybe a point to have some uniformity?